spec/policies/defence_request_controller_policy_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe DefenceRequestControllerPolicy do
+  subject { DefenceRequestControllerPolicy }
+
+  [:index?].each do |action|
+    permissions action do
+
+      it "grants access to #{action} if user role is 'cso'" do
+        expect(subject).to permit(User.new(role: :cso), DefenceRequestControllerPolicy.new)
```

Comment:
It the second argument right? From the docs it's meant to be an model or something that can have an action performed on it.